mkosi

```diff
@@ -2560,6 +2583,7 @@ def parse_args() -> CommandLineArguments:
     group.add_argument("--build-dir", help='Path to use as persistent build directory', metavar='PATH')
     group.add_argument("--build-package", action=CommaDelimitedListAction, dest='build_packages', default=[], help='Additional packages needed for build script', metavar='PACKAGE')
     group.add_argument("--postinst-script", help='Postinstall script to run inside image', metavar='PATH')
+    group.add_argument("--postinst2-script", help='Postinstall script to run outside image', metavar='PATH')
```

Comment:
I don't know if there is any precedent for naming thing like that but perhaps `--postinst-outer-script` would be more direct, the `2` is a bit magic (granted the help message is clear and explicit).